### PR TITLE
fix(@desktop/wallet): estimated time for transaction is always Unknown

### LINKED ISSUE #5398: Estimated time for transaction is always Unknown
### Description

Screenshot 2022-04-06 at 15 50 20

```
property string estimatedTime: qsTr("Unknown")
```

No reason to keep it as a string. We need to either add a logic to fetch the estimation from transaction details, or remove the label at all

### LINKED ISSUE #5938: Right click - quit app makes the app not responding
### Description

Cant quit the app by clicking right click on the icon and quit - the app hangs and is not responsive

**Steps:**

- open app
- login
- right click - quit

Screenshot 2022-05-31 at 11 17 09

### PR DESCRIPTION
**Pull request by saledjenic**
Fixes #5398

In general applied logic rely on previous block and where would this transaction fall if it was included in the previous block.

https://user-images.githubusercontent.com/86303051/171695501-face2bf6-7fd0-4cf2-b6dc-31e0be8e0b44.mov

### CODE REVIEW

File: src/app_service/service/transaction/service.nim [hunk 1]
```diff
@@ -362,3 +369,69 @@ QtObject:
...
+
+  proc getEstimatedTime*(self: Service, priorityFeePerGas: string, maxFeePerGas: string): EstimatedTime = 
+    let priorityFeePerGasF = priorityFeePerGas.parseFloat
+    let maxFeePerGasF = maxFeePerGas.parseFloat
+    var transactionsProcessed = 0
+    var numOfTransactionWithTipLessThanMine = 0
+    var latestBlockNumber: Option[Uint256]
+    var expectedBaseFeeForNextBlock: float
+    try:
+      let response = eth.getBlockByNumber("latest", true)
```
Comment (richard-ramos) [inline src/app_service/service/transaction/service.nim:391]: I think just taking in account the latest block might not reflect an accurate result. Maybe during QA we can compare the results of this formula against these websites, to know how close are we to their estimates?
https://www.gasprice.io/
https://ethgasstation.info/
https://etherscan.io/gastracker

Comment (saledjenic) [reply src/app_service/service/transaction/service.nim:391]: @richard-ramos it doesn't take only the last block, but last ~100 transactions, that can be last 5,6,7... 10... blocks.

Comment (alaibe) [reply src/app_service/service/transaction/service.nim:391]: You can have a look to the method: eth fee history to see more block details:
e.g: to calculate gas price suggestion we do:
```
err = f.RPCClient.Call(&feeHistory, chainID, "eth_feeHistory", 101, "latest", nil)
    if err != nil {
        return nil, err
    }
```

**Comment by alaibe** [comment]
What will happen for chain that have not yet enabled EIP1559?

**Review by anastasiyaig — APPROVED** [review]

**Comment by anastasiyaig** [comment]
https://ethstatus.testrail.net/index.php?/runs/view/9365

**Pull request merged**